### PR TITLE
Update page template & global styles

### PR DESCRIPTION
**Pull request by richardmatthewsdev**
**Acceptance Criteria**

-Page template implemented across app
-Nav/colours/fonts universal

### CODE REVIEW

**Review by paul-mesnilgrente — APPROVED** [review]

File: app/frontend/entrypoints/application.js [hunk 1]
```diff
@@ -36,6 +36,8 @@ import "/js/TestRecordExtraction";
 import "/js/TestEnrichmentExtraction";
 import "/js/TestTransformationRecordSelector";
 import "/js/TestDestination";
+import "/js/ContentSourceFilter.js";
```
Comment (isabel-anastasiadis-boost) [inline app/frontend/entrypoints/application.js:39]: Do you need the .js?  Super minor, just makes it consistent

File: app/views/destinations/edit.html.erb [hunk 2]
```diff
@@ -1,9 +1,21 @@
 <%= content_for(:title) { "Editing #{@destination.name_in_database}" } %>
 
-<%= render 'shared/breadcrumb_nav' %>
+<%= content_for :header do %>
+  <%= render 'shared/breadcrumb_nav' %>
 
-<h1><%= @destination.name_in_database %></h1>
+  <h1 class="header__title"><%= @destination.name_in_database %></h1>
```
Comment (isabel-anastasiadis-boost) [inline app/views/destinations/edit.html.erb:6]: I'm a bit confused, is `header` a block that this is an element of? I don't see the `header` class here

Comment (richardmatthewsdev) [reply app/views/destinations/edit.html.erb:6]: Hey @isabel-anastasiadis-boost! Yes the title is an element inside the header. The header is on the application.html.erb where the yield :header is :)

File: app/frontend/entrypoints/application.scss [hunk 3]
```diff
@@ -1,16 +1,35 @@
...
 
 // Blocks
-@import "../stylesheets/blocks/icons";
+
+@import "../stylesheets/blocks/card";
+@import "../stylesheets/blocks/form-in-dropdown";
+@import "../stylesheets/blocks/header";
 @import "../stylesheets/blocks/record-view";
 
+
```
Comment (isabel-anastasiadis-boost) [inline app/frontend/entrypoints/application.scss:26]: Maybe consider moving this to its own file

File: app/frontend/js/TestDestination.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { bindTestForm } from './utils/test-form';
 
-bindTestForm('test', 'js-test-destination', (response, alertClass) => {
+bindTestForm('test', 'js-test-destination', 'js-destination-form', (response, alertClass) => {
```
Comment (isabel-anastasiadis-boost) [inline app/frontend/js/TestDestination.js:3]: Maybe could add -button to the end of the `js-test-destination` id

File: app/frontend/js/form-header-submission.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+const contentSourceFormSubmit = document.getElementById('js-content-source-form-submit');
```
Comment (isabel-anastasiadis-boost) [inline app/frontend/js/form-header-submission.js:1]: Might be good to add a comment explaining what this is for /why we need it

File: app/frontend/js/form-header-submission.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+if(contentSourceFormSubmit) {
+  contentSourceFormSubmit.addEventListener("click", (event) => {
+    document.getElementById('js-content-source-form').submit();
+  });
+}
+
+const extractionFormSubmit = document.getElementById('js-extraction-definition-form-submit');
```
Comment (isabel-anastasiadis-boost) [inline app/frontend/js/form-header-submission.js:9]: Looks like lots of dulication - code climate might not be happy

File: app/frontend/stylesheets/blocks/_header.scss [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    float: left;
+    margin-bottom: 0;
+  }
+
+  @include element('actions') {
+    float: right;
+    padding-top: .5rem;
+  }
+
+  .nav-tabs {
```
Comment (isabel-anastasiadis-boost) [inline app/frontend/stylesheets/blocks/_header.scss:17]: Consider adding bem classes for these if we can

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -10,23 +10,20 @@
...
 
     <%= render 'shared/nav' %>
 
+    <header class='header'>
+      <div class='container'>
+        <%= yield :header %>
+      </div>
+    </header>
+
     <div class="<%= content_for?(:root_class) ? yield(:root_class) : 'container' %>">
```
Comment (isabel-anastasiadis-boost) [inline app/views/layouts/application.html.erb:25]: Lets remove if we don't need it :)

**Review by isabel-anastasiadis-boost — APPROVED** [review]
Looks good!

File: app/frontend/entrypoints/application.js (hunk 1 above)
Comment (richardmatthewsdev) [reply app/frontend/entrypoints/application.js:39]: Yeah we don't need the js haha 👯

File: app/frontend/entrypoints/application.scss (hunk 3 above)
Comment (richardmatthewsdev) [reply app/frontend/entrypoints/application.scss:26]: Done :)

File: app/frontend/js/TestDestination.js (hunk 4 above)
Comment (richardmatthewsdev) [reply app/frontend/js/TestDestination.js:3]: all done :)

File: app/frontend/js/form-header-submission.js (hunk 6 above)
Comment (richardmatthewsdev) [reply app/frontend/js/form-header-submission.js:9]: Fair call! Iv updated this file to be dynamic and all you need now is an id on your form and a data-form attribute on your button.

**Pull request merged**